src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -114,9 +133,15 @@ export const NavBarSchedule: FC<NavBarScheduleProps> = ({ isVisible }) => {
   );
 
   const schedule: ScheduleDay = useMemo(() => {
-    const dayStart = addDays(startOfToday(), selectedDayIndex);
+    const dayStart = addDays(firstDay, selectedDayIndex);
```

Comment:
```suggestion
    const dayStart = addDays(firstDayOfSchedule, selectedDayIndex);
```

Reply:
Done

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1419/commits/b78373170c83bbc19324c8694b04ea35b9ab06ef